include/swift/AST/GenericSignatureBuilder.h

```diff
@@ -936,9 +936,11 @@ class GenericSignatureBuilder::RequirementSource final
   /// Whether there is a trailing written requirement location.
   const bool hasTrailingWrittenRequirementLoc;
 
+public:
```

Comment:
Does this need to be public?

Reply:
Yeah, it's used from a static function. It's `const`, so there's no concern about anyone messing with the state.

Reply:
... although this makes me wonder if I could back-patch the value when it *is* found to map into the requirement signature, to prevent future work. Feels icky.